app/scripts/modules/core/healthCounts/HealthCounts.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+import { isEqual } from 'lodash';
+
+import { InstanceCounts } from 'core/domain';
+import { Tooltip } from 'core/presentation/Tooltip';
+
+import './healthCounts.less';
+
+interface IProps {
+  container: InstanceCounts;
+  additionalLegendText?: string;
+  legendPlacement?: 'top' | 'bottom' | 'left' | 'right';
```

Comment:
I have the same list for popover components. We should extract a typescript type for `'top' | 'bottom' | 'left' | 'right'`